cmd/mdatagen/loader.go

```diff
@@ -115,7 +115,7 @@ type templateContext struct {
 }
 
 func loadMetadata(filePath string) (metadata, error) {
```

Comment:
I think it is cleaner if `file:` is added inside `loadMetadata` instead of when you call it, otherwise the `filePath` argument name is misleading.

Reply:
good catch, updated 👍